packages/react/src/file-input/FileInput.tsx

```diff
@@ -127,3 +154,9 @@ function useFileInputStyles({ hasFile }: { hasFile: boolean }) {
 		'&:focus': packs.outline,
 	};
 }
+
+const hasLabel = (
```

Comment:
Man, I dislike TS. Feels so heavy handed

Reply:
I agree with you on this one!